apis/topology/v1beta1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2023.
```

Comment:
nit: 2024 or maybe 2025

Reply:
+1 updating, I guess it comes from some automatic scaffolding of the version I have locally.

Reply:
Done